docs/src/components/ComponentDoc/ComponentControls/ComponentControls.tsx

```diff
@@ -18,7 +18,10 @@ type ComponentControlsProps = {
   onShowRtl: (e: React.SyntheticEvent) => void
   onShowTransparent: (e: React.SyntheticEvent) => void
   onShowVariables: (e: React.SyntheticEvent) => void
+  showCode: boolean
   showRtl: boolean
+  showVariables: boolean
```

Comment:
Not sure if we should add the `showVariabels` in this moment, until we have a way to share it's values...